.gitignore

```
@@ -0,0 +1,6 @@
+**/*.tfstate
+**/*.tfplan
+**/*.tfstate.backup
+**/.terraform
```

I think this should work?
```suggestion
.terraform/
```